models/intermediate/int_user_onboarding.sql

```diff
@@ -0,0 +1,27 @@
+{% set field_names_query %}
+    SELECT DISTINCT field_name FROM {{ ref('stg_user_attributes') }}
+{% endset %}
+
+with
+    user_attributes_cte as (

```

Comment:
what does `cte` means here?

Reply:
CTE means Common Table Expression but here it doesn't make any sense so we can remove.